src/CoDa.jl

```diff
@@ -26,6 +27,13 @@ export
   norm, dot, ⋅,
   distance,
 
+  # matrices
+  JColumn,
+  JMatrix,
```

Comment:
Because of the conflicting styles - structures should start w/ upper case letters according to the [manual](https://docs.julialang.org/en/v1/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/).

There're is also the problem of `j` beeing a common indexation variable.

Reply:
Do you have any suggestions for `j` alternatives? Maybe `Jc`?

Reply:
Maybe for `Julia` this doesn't apply, but the variables I use more frequently in nested `for` loops are `i`, `j` and `k`. Wouldn't `j` for the column vector create confusion?